app/src/main/java/app/revanced/integrations/youtube/patches/spoof/SpoofVideoStreamsPatch.java

```diff
@@ -69,9 +69,9 @@ public static String blockInitPlaybackRequest(String originalUrlString) {
                 String path = originalUri.getPath();
 
                 if (path != null && path.contains("initplayback")) {
-                    Logger.printDebug(() -> "Blocking 'initplayback' by returning unreachable url");
+                    Logger.printDebug(() -> "Blocking 'initplayback' by clearing query");
```

Comment:
This would still request YouTube. Why not return an unreachable URL?

Reply:
What fixes it is using the original request headers. They may expire, so they have to be refreshed; this PR handles it by using the original request headers, so they are always up to date. Blocking the initplayback request by clearing the query should not have any effect on this. An unreachable URL should be used to avoid bothering YouTube with error requests.